operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java

```diff
@@ -445,27 +504,32 @@ private Step createEventSteps(List<EventData> eventDataList) {
     List<EventData> createDomainEvents() {
       List<EventData> list = new ArrayList<>();
       if (hasJustExceededMaxRetryCount()) {
-        list.add(new EventData(DOMAIN_PROCESSING_ABORTED).message(INTROSPECTOR_MAX_ERRORS_EXCEEDED));
+        list.add(
+            new EventData(EventHelper.EventItem.DOMAIN_FAILED)
+                .failureReason(Aborted)
+                .message(INTROSPECTOR_MAX_ERRORS_EXCEEDED));
       } else if (hasJustGotFatalIntrospectorError()) {
-        list.add(new EventData(DOMAIN_PROCESSING_ABORTED)
-              .message(FATAL_INTROSPECTOR_ERROR_MSG + getNewStatus().getMessage()));
+        list.add(new EventData(EventHelper.EventItem.DOMAIN_FAILED)
+            .failureReason(Aborted)
+            .message(FATAL_INTROSPECTOR_ERROR_MSG + getNewStatus().getMessage()));
       }
       return list;
     }
 
     private boolean hasJustGotFatalIntrospectorError() {
       return isFatalIntrospectorMessage(getNewStatus().getMessage())
-            && !isFatalIntrospectorMessage(getStatus().getMessage());
+          && !isFatalIntrospectorMessage(getStatus().getMessage());
     }
 
     private boolean isFatalIntrospectorMessage(String statusMessage) {
       return statusMessage != null && statusMessage.contains(FATAL_INTROSPECTOR_ERROR);
     }
 
     private boolean hasJustExceededMaxRetryCount() {
+      int jobFailureCount = getNewStatus().getIntrospectJobFailureCount();
```

Comment:
should this be renamed "hasJustReachedMaxRetryCount()" ?

Reply:
I think the name matches the functionality and the name is not new in this PR.

Reply:
Oops, you're right of course. Misread the diff.  Resolving.